src/test/java/core/basesyntax/ConcatenatedWordsTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConcatenatedWordsTest {
+    private final static String LONGEST_CONCATENATED_WORD = "ratcatdogcat";
+    private final static String SECOND_LONGEST_CONCATENATED_WORD_WHEN_SAME_LENGTH = "catdogcatrat";
+    private final static String SECOND_LONGEST_CONCATENATED_WORD = "catsdogcats";
+    private final static String NO_CONCATENATED_WORD = "";
+    private final static Integer TOTAL_COUNT_OF_CONCATENATED_WORDS = 3;
+    private final static String[] ARRAY_OF_WORDS = {"cat", "cats", "catsdogcats",
```

Comment:
```suggestion
    private final static List<String> letsAddVariableNameInCamelCase = List.of("cat", "cats", "catsdogcats", ...
```
replace all arrays with list of

Reply:
What about this? @karsv

Reply:
propustil ocen` stidno... I'm sorry! And i fixed< and it's explained List.of()...